src/resources/help/en_US/relnotes3.3.0.html

```diff
@@ -74,6 +74,7 @@
   <li>Custom titles for Legend annotation objects can now be set.</li> 
   <li>Dolly and FlyThrough navigation modes now support mouse-wheel interaction for zooming.</li>
   <li>Changing navigation mode (Trackball, Dolly, FlyThrough) will now trigger an automatic View reset.</li>
+  <li>The Color Table Window and Buttons have been redesigned. In short, color table categories have been removed, and replaced with a new tagging system. Color tables have a number of tags associated with them, and when tag filtering is enabled, users can select multiple tags to see only color tables associated with those tags. There are additional options for how tag selection affects the visible results.</li>
```

Comment:
I proposed to replace `visible` here to avoid confusion with visualization results...

```suggestion
  <li>The Color Table Window and Buttons have been redesigned. In short, color table categories have been removed, and replaced with a new tagging system. Color tables have a number of tags associated with them, and when tag filtering is enabled, users can select multiple tags to see only color tables associated with those tags. There are additional options for how tag selection affects the resulting matching list of ColorTables.</li>
```